.credo.exs

```diff
@@ -0,0 +1,32 @@
+# config/.credo.exs
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        #
+        # You can give explicit globs or simply directories.
+        # In the latter case `**/*.{ex,exs}` will be used.
+        #
+        included: [
+          "lib/"
+        ],
+        excluded: [~r"/_build/", ~r"/deps/", ~r"/node_modules/"]
+      },
+      checks: [
+        {Credo.Check.Refactor.MapInto, false},
+        {Credo.Check.Warning.LazyLogging, false},
+        {Credo.Check.Readability.LargeNumbers, only_greater_than: 86400},
```

Comment:
You have only greater than 86400 here, but I noticed that you put underscores in quite a few smaller numbers. Wasn't sure whether you wanted this or not.

Reply:
Good catch, having that underscore in ports feels odd to me so I reverted. I think I wasn't using this config yet.